students/Vincent_Aquila/session03/list_lab.py

```diff
@@ -0,0 +1,185 @@
+"""
+Python210A Vincent Aquila  Fall2018
+using Python 3.7.0
+assignment objective: List Lab
+
+Note - I am using many/extra print statements in the code for the purpose of
+testing.  Comment and Doc Strings might be explaining the obvious to experienced
+developers; if they seem redundant, it is for my learning purposes.
+
+There are four sections, each beginning with #Series 1, #Series 2, etc.
+"""
+
+
+
+#Series 1
+"""
+-Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+-Display the list (plain old print() is fine…).
+-Ask the user for another fruit and add it to the end of the list.
+-Display the list.
+-Ask the user for a number and display the number back to the user and the fruit
+corresponding to that number (on a 1-is-first basis). Remember that Python uses
+zero-based indexing, so you will need to correct.
+-Add another fruit to the beginning of the list using “+” and display the list.
+-Add another fruit to the beginning of the list using insert() and display the
+list.
+-Display all the fruits that begin with “P”, using a for loop.
+"""
+
+list = ["Apples", "Pears", "Oranges", "Peaches"]
+for i in list:
+    print(i)
+"""
+list = ["Apples", "Pears", "Oranges", "Peaches"] - *note to self* the list
+positioned below a for loop throws a TypeError - list needs to be above for loop
+"""
+response = input("Please select one other fruit you want added to this list. > ")
+new_fruit = response.title()
+"""
+The purpose of .title() is to ensure the first letter of the word is capitalized.
+Later in the program, in a for loop, the loop searches for words, and those words
+begin with a capital letter.  If a user entered a word, where the first letters
+is not capitalized, then the program would not work properly.  .title() ensures
+the program works correctly.
+"""
+#print(new_fruit) #test point to see the new fruit that was requested
+
+list.append(new_fruit)
+print(list) #this prints the amended list with the new fruit added to the end
+
+response = input("Please pick a number between 0 and 4. > ")
+number = response
+print(number)
+#note, further work could be done - when entering numbers outside 0-4
+
+for j, item in enumerate(list): #generates two columns: numbers and fruits
+    print(j, item)
+print(list)
+
+#add second fruit to the beginning of the list using +
+list.insert(0, "Strawberry")
+print(list)
+"""
+I learned a lot here - tried to use:
+    l = (str(k)[1:-1])
+which removes brackets from the list in the terminal display - but - after
+stripping brackets from a list and then employing an insert method -
+it won't work - throws an attribute error.  Insert only works with a bracketed
+list.  Unable to use the + sign to add the second fruit to the beginning of the
+list.
+"""
+
+#add a thrid fruit to the beginning of the list using insert()
+list.insert(0, "Kiwi")
+print(list)
+
+#display all the friuts that begin with "P" using a for loop
+for item in list:
+	if "P" in item:
```

Comment:
you only want the the ones that begin with "P" -- so not APPLE -- granted, if your capitalization is normalized, this will work, but it's not really the right test...